new-dti-website/components/course/DDProjects.tsx

```diff
@@ -30,8 +30,8 @@ export default function DDProjects({ title, description, imageSrc }: DDProjectsP
   };
 
   return (
-    <div
-      className={`transition-all duration-300 ease-in-out ${
+    <button
```

Comment:
I've noticed that these FAQ accordion elements actually have two focus states: one for the whole element and one for the icon. I think the icon should be purely visual (ie: you shouldn't be able to interact/focus on it). 


 
<img width="1274" alt="accordgion focus state" src="https://github.com/user-attachments/assets/d53cd9b3-8288-4c09-94a0-18050a37593c">
<img width="1254" alt="inner focus state" src="https://github.com/user-attachments/assets/e9aa06db-f285-4d75-b9b6-ddff37e145e3">


Nested buttons is also typically bad practice because it's not semantic HTML and it can mess up screen readers, so i think these are other reasons we should remove the inner `<button>` element containing the icon.